tests/test_enforce.py

```diff
@@ -37,7 +38,7 @@ class Subclazz(Enforcing):
             classVariableIsOk = "OK!"
 
             @overrides
-            def nonfinal1(self, param: int) -> int:
```

Comment:
Bug detected using signature validation; the return type does not match that of the inherited method.

Reply:
Great! Mypy would most likely then resolve the problem with the actual return value being non string!